### PR TITLE
Add Lombok support

### LINKED ISSUE #56: Add Lombok Support

### PR DESCRIPTION
**Pull request by AriSengupta**
Description:
- Add Lombok dependency
- Convert classes to use Lombok annotations
- Move dtos to separate package

Testing:
- All existing test cases pass

Closes #56

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -99,6 +99,12 @@
             <version>30.1.1-jre</version>
         </dependency>
 
+        <dependency>
+            <groupId>org.projectlombok</groupId>
+            <artifactId>lombok</artifactId>
+            <version>1.18.16</version>
```
Comment (cyang030) [inline pom.xml]: maybe grab the latest v1.18.20?

File: src/test/java/application/models/TestAnswerDTO.java [hunk 2]
```diff
@@ -1,5 +1,6 @@
 package application.models;
 
+import application.models.dto.AnswerDTO;
```
Comment (cyang030) [inline src/test/java/application/models/TestAnswerDTO.java]: How about moving the DTO tests into similar structure?  test/java/application/models/dto/TestAnswerDTO.java?

File: pom.xml (hunk 1 above)
Comment (AriSengupta) [reply pom.xml]: Fixed

File: src/test/java/application/models/TestAnswerDTO.java (hunk 2 above)
Comment (AriSengupta) [reply src/test/java/application/models/TestAnswerDTO.java]: Good catch. Fixed.

**Review by cyang030 — APPROVED** [review]

**Review by thomas7carriere — APPROVED** [review]
looks good to me

**Pull request merged**